cmd/gitbase/command/server.go

```diff
@@ -395,12 +421,19 @@ func parseDirectory(dir directory) (directory, error) {
 			dir.Format = val
 
 		case "bare":
-			if val != "true" && val != "false" {
+			if val != "true" && val != "false" && val != "auto" {
```

Comment:
why do we need this `if`, if we can add a `default` to the `switch`?

Reply:
Totally right. I modified the previous option that only had two possibilities. :+1:

Reply:
Changed